target/bin/fail2ban

```diff
@@ -3,6 +3,23 @@
 # shellcheck source=../scripts/helper-functions.sh
 . /usr/local/bin/helper-functions.sh
 
+if ! IPTABLES_OUTPUT=$(iptables -L 2>&1)
+then
+  echo "IPTables does not seem to work properly. The output of 'iptables -L' was:
+
+${IPTABLES_OUTPUT}
+
+Possible causes for this errore are
+
+1. Missing capabilities
+2. Modification by user-patches.sh
+
+Aborting.
```

Comment:
Usually 2 or 3 dots for an output implying an action being taken? Either that or no dots.

```suggestion
Aborting..
```

Reply:
Done. [334898a8a79b16c85984e02d8c5506f5f9792e0c]